directory_sso_api_client/user.py

```diff
@@ -44,3 +45,7 @@ def get_last_login(self, start=None, end=None):
             url=self.endpoints['last_login'],
             params=params
         )
+
+    def get_user_by_email(self, email):
+        url = self.endpoints['user_by_email'] % email
```

Comment:
please use the `.format()` approach, was we're consistently using that approach everywhere else as far as I can tell.